tests/integration/test_rex.py

```diff
@@ -556,32 +553,32 @@ def test_date_functions(c):
             EXTRACT(QUARTER FROM d) AS "quarter",
             EXTRACT(SECOND FROM d) AS "second",
             EXTRACT(WEEK FROM d) AS "week",
-            EXTRACT(YEAR FROM d) AS "year"
-
-            LAST_DAY(d) as "last_day",
-
-            TIMESTAMPADD(YEAR, 2, d) as "plus_1_year",
-            TIMESTAMPADD(MONTH, 1, d) as "plus_1_month",
-            TIMESTAMPADD(WEEK, 1, d) as "plus_1_week",
-            TIMESTAMPADD(DAY, 1, d) as "plus_1_day",
-            TIMESTAMPADD(HOUR, 1, d) as "plus_1_hour",
-            TIMESTAMPADD(MINUTE, 1, d) as "plus_1_min",
-            TIMESTAMPADD(SECOND, 1, d) as "plus_1_sec",
-            TIMESTAMPADD(MICROSECOND, 999*1000, d) as "plus_999_millisec",
-            TIMESTAMPADD(MICROSECOND, 999, d) as "plus_999_microsec",
-            TIMESTAMPADD(QUARTER, 1, d) as "plus_1_qt",
-
-            CEIL(d TO DAY) as ceil_to_day,
-            CEIL(d TO HOUR) as ceil_to_hour,
-            CEIL(d TO MINUTE) as ceil_to_minute,
-            CEIL(d TO SECOND) as ceil_to_seconds,
-            CEIL(d TO MILLISECOND) as ceil_to_millisec,
-
-            FLOOR(d TO DAY) as floor_to_day,
-            FLOOR(d TO HOUR) as floor_to_hour,
-            FLOOR(d TO MINUTE) as floor_to_minute,
-            FLOOR(d TO SECOND) as floor_to_seconds,
-            FLOOR(d TO MILLISECOND) as floor_to_millisec
+            EXTRACT(YEAR FROM d) AS "year",
+
+            LAST_DAY(d) as "last_day"
+
+            -- TIMESTAMPADD(YEAR, 2, d) as "plus_1_year",
+            -- TIMESTAMPADD(MONTH, 1, d) as "plus_1_month",
+            -- TIMESTAMPADD(WEEK, 1, d) as "plus_1_week",
+            -- TIMESTAMPADD(DAY, 1, d) as "plus_1_day",
+            -- TIMESTAMPADD(HOUR, 1, d) as "plus_1_hour",
+            -- TIMESTAMPADD(MINUTE, 1, d) as "plus_1_min",
+            -- TIMESTAMPADD(SECOND, 1, d) as "plus_1_sec",
+            -- TIMESTAMPADD(MICROSECOND, 999*1000, d) as "plus_999_millisec",
+            -- TIMESTAMPADD(MICROSECOND, 999, d) as "plus_999_microsec",
+            -- TIMESTAMPADD(QUARTER, 1, d) as "plus_1_qt",
+
+            -- CEIL(d TO DAY) as ceil_to_day,
```

Comment:
Waiting on https://github.com/sqlparser-rs/sqlparser-rs/pull/635

Reply:
@sarahyurick is there a reference to this format `<expr> TO <type>`, please? I can't find anything about that

Reply:
I'm having trouble finding it myself. For the Dask-SQL project, it was first implemented with #91, although not much context was provided.

Reply:
@sarahyurick I'll keep looking but unless there's a source where we can find it, I don't think it's implementable at all.

It looks like the `CAST` from standard, but with smaller syntax, but I'm not quite sure as it seems really specific to a random dialect or something like that.